src/plugins/visualize/public/application/components/visualize_byvalue_editor.tsx

```diff
@@ -33,6 +33,7 @@ import {
 import { VisualizeServices } from '../types';
 import { VisualizeEditorCommon } from './visualize_editor_common';
 import { VisualizeAppProps } from '../app';
+import { VisualizeConstants } from '../..';
```

Comment:
You could also `import { VisualizeConstants } from '../visualize_constants';` for readability but this is just a nit